labs/import_data.ipynb

```diff
@@ -16,7 +16,7 @@
     "\n",
     "If you have not completed the [Download data lab](https://developers.arcgis.com/labs/python/download-data), the code in the **The Download Data Lab** cell immediately below must be run before the **Import Data Lab** cells will run. These cells contain the code in the ArcGIS API for Python [Download data lab solution](download_data.ipynb). The LA_Hub_Datasets.zip directory from which you import data is only available in a single notebook.esri.com session.\n",
     "\n",
-    "If you have completed the [Download data lab](https://developers.arcgis.com/labs/python/download-data) and are still in that notebooks.esri.com session, skip to [The Import Data Lab](#import_section) section of this lab."
+    "If you have completed the [Download data lab](https://developers.arcgis.com/labs/python/download-data) and are still in that notebook.esri.com session, skip to [The Import Data Lab](#import_section) section of this lab."
```

Comment:
notebooks.esri.com is the correct URL, not notebook.esri.com. It should probably say "are still in that notebook, skip to..."

Reply:
"notebook.esri.com" is referred to twice in the note.